src/pages/about.module.css

```diff
@@ -0,0 +1,23 @@
+@import "../styles/settings.css";
+
+.root {
+  max-width: $ten-columns;
+  margin: 0 auto;
+}
+
+.section1 {
```

Comment:
🙈

Reply:
Man, I can't even xD also matches the stuff on the CMS sorry